build.sbt

```diff
@@ -32,9 +32,7 @@ lazy val jsonBench = (project in file("json-bench"))
     name := "json-bench",
     mainClass := Some("com.github.saint1991.samples.JsonBench"),
     libraryDependencies ++= Seq(
```

Comment:
Could you leave circe benchmark besides your library's one?
It is good information to show users that your library is faster than circe, I think.

Reply:
Another point to exclude circe is that is usually used to parse/serialize from/to string that is error prone.
Sometime users miss to set a right charset (usually "UTF-8") when converting strings to/from bytes. Also, by default JDKs byte array to string decoders silently replace malformed bytes by placeholder character, or JDKs string to byte array encoders never flag an error in case of broken surrogate pair.
Jsoniter-scala will throw an exception in all these cases.

Reply:
Sorry, my words were misleading.
I got that your library is faster than Circe and has better API that handles bytes.
However, Circe is also widely used JSON library in Scala so I want to keep its' benchmark result here.

Cloud you write the benchmark against your library as a different subproject?
I mean `json-bench/circe/src/main` for circe benchmark, `json-bench/jsoniter/src/main` for jsonier-scala. 
If it is not convenient for you, merge this PR first and I will rewrite Circe benchmark as stated above.
What do you think about that?

Reply:
np, I will do as you have proposed

Reply:
@saint1991 I done it... please check latest commit https://github.com/saint1991/serialization-benchmark/pull/2/commits/956f6334533dfa0b0b97d184b51c0785e2c4f453